Gemfile

```diff
@@ -1,11 +1,17 @@
 # encoding: utf-8
 source 'https://rubygems.org'
 
-gemspec name: 'inspec'
```

Comment:
Isn't gemspec the proper way to source the local gem or do we have to do this here?

Reply:
this was necessary, although i can't quite recall what blew up now.  copied this from the chef/chef Gemfile.

Reply:
Note that we have two gemspecs, because we build two gems - inspec and inspec-core.  I think this line was there mainly to pass along the gemspec name explicitly.

Reply:
`gem "inspec", path: "."` only picks up the inspec gem out of the inspec.gemspec though.  the name is there explicitly.

Reply:
Perhaps I'm misunderstanding, but we also need to be able to build `inspec-core`, which has a separate gemspec (and different (lighter) dependencies). Would that still work?

Reply:
You don't build that for including into the Gemfile though right?  Because you weren't including `inspec-core` here before either...

Reply:
The Gemfile shouldn't interfere with inspec-core builds as we're building it with `gem build`.